Dockerfile

```diff
@@ -15,6 +16,7 @@ RUN JOBS=MAX npm ci --unsafe-perm --production && npm cache clean --force && rm
 
 # Copy and enable the service
 COPY config/services /etc/systemd/system
+RUN systemctl disable dnsmasq.service
```

Comment:
If we _never_ want dnsmasq running here, it might be _slightly_ safer to `mask` the unit (preventing anything else from pulling in dnsmasq as a dep)